docs/README.md

```diff
@@ -12,14 +12,23 @@
 [![npm version](https://badge.fury.io/js/xstate.svg)](https://badge.fury.io/js/xstate)
 <img src="https://opencollective.com/xstate/tiers/backer/badge.svg?label=sponsors&color=brightgreen" />
 
-JavaScript and TypeScript [finite state machines](https://en.wikipedia.org/wiki/Finite-state_machine) and [statecharts](https://www.sciencedirect.com/science/article/pii/0167642387900359/pdf) for the modern web.
+Handle complexity anywhere JavaScript runs using statecharts.
 
-New to state machines and statecharts? [Read our introduction](/guides/introduction-to-state-machines-and-statecharts/).
+- Create **robust applications**, both on the **frontend and backend**
+- **Visualize your app's logic** in our awesome **dev tools**
+- Integrates with Node, [React](./packages/xstate-react/index.md), [Vue](./packages/xstate-vue/index.md), [Svelte](./packages/xstate-svelte/index.md) - anywhere JavaScript runs
```

Comment:
Yyyy... Unsure, we probably need a page for how to implement XState in the backend. But since there's no strong recommendations/integration package, it's a little dicey.